docs/service-brokers/gcp-broker-service-classes/cloud-bigtable/docs/plans-details.md

```diff
@@ -0,0 +1,79 @@
+| **type** | `string` | The value of this parameter can be either `PRODUCTION` or `DEVELOPMENT`. If your Cloud Bigtable cluster serves data to production, choose `Production`. If you want to experiment with Bigtable without committing to a production-grade cluster, choose `Development`. However, no Service Level Agreement (SLA) applies. | NO | `PRODUCTION` |
+
+### Cluster properties
+
+These are the properties that you can set for your Cloud Bigtable cluster:
+
+| Parameter Name | Type | Description | Required | Default Value |
+|----------------|------|-------------|----------|---------------|
+| **clusterId** | `string` | Unique and permanent identifier for Cloud Bigtable instance. Use only lowercase letters, numbers, or hyphens. Must be 6-30 characters long. | YES | - |
+| **defaultStorageType** | `string` | Storage type affects Node performance and monthly storage costs. The value of this parameter can be either `SSD` or `HDD`. The choice is permanent. | YES | `SSD` |
+| **location** | `string` |  Determines where Cloud Bigtable data is stored. To reduce latency and increase throughput, store your data near the services that need it. These are the possible values: `us-east1-b`, `us-east1-c`, `asia-east1-b`, `asia-east1-a`, `asia-northeast1-c`, `asia-northeast1-b`, `europe-west1-b`, `europe-west1-c`, `europe-west4-b`, `europe-west1-d`, `us-central1-c`, `us-central1-b`, `us-central1-f`, `asia-southeast1-b`. The choice is permanent.  | YES | - |
+| **serveNodes** | `string` | Add Nodes to increase capacity for data throughput and queries per second (QPS). Only applies to `PRODUCTION`. The minimal number of Nodes is 3. | NO | - |
+
+
+### Tables properties
+
+These are the **Tables** properties:
+
+| Parameter Name | Type | Description | Required | Default Value |
+|----------------|------|-------------|----------|---------------|
+| **tableId** | `string` |  The name by which you refer to the new table within the parent instance. | YES | - |
+| **columnFamily** | `array` | A set of columns within a table which share a common configuration. | NO | - |
+| **ColumnFamily.columnFamilyId** | `string` | The name of the ColumnFamily. | NO | - |
+| **ColumnFamily.gcRule** | `object` |  Rule used to determine which cells to delete during garbage collection. Must serialize to at most 500 bytes. | NO | - |
+| **ColumnFamily.gcRule.maxAge** | `string` |  Deletes cells in a column older than a given age. Values must be at least one millisecond, and are truncated to microsecond granularity. | NO | - |
+| **ColumnFamily.gcRule.maxNumVersions** | `integer` |  Deletes all cells in a column, except the most recent. | NO | - |
+| **granularity** | `string` |  The granularity at which timestamps are stored in this table. Timestamps not matching the granularity are rejected. The value of this parameter is `MILLIS`. | NO | `MILLIS` |
```

Comment:
If the default value is already provided in the last column, do we need the sentence saying: 
```
The value of this parameter is `MILLIS`

```
🤔